Services/Component/classes/class.ilPluginSlot.php

```diff
@@ -208,8 +208,11 @@ function getPrefix()
 	*/
 	function getPluginsInformation()
 	{
-		global $DIC;
-		$ilPluginAdmin = $DIC['ilPluginAdmin'];
+		global $ilPluginAdmin;
+		// $DIC does not work here, db step <#5088> will fail, since initiliases the global
+		// object, but not the DIC object
+		//global $DIC;
+		//$ilPluginAdmin = $DIC['ilPluginAdmin'];
```

Comment:
Is this really related to the KS-Stuff?

Reply:
No, but this makes the branch installable, see https://www.ilias.de/mantis/view.php?id=23163
I will remove this before merging.